libtaxii/__init__.py

```diff
@@ -52,8 +52,13 @@ def get_message_from_http_response(http_response, in_response_to):
     else:
         raise ValueError('Unsupported response type: %s.' % http_response.__class__.__name__)
 
+def return_check(return_message,message,object):
```

Comment:
That is what I suggested [here](https://github.com/TAXIIProject/libtaxii/pull/213#issuecomment-256759292), and I maintain that it is a better approach than the `return_check` function.